examples/pytorch/gcn/gcn_mp.py

```diff
@@ -1,36 +1,24 @@
-"""GCN using basic message passing
-
-References:
-- Semi-Supervised Classification with Graph Convolutional Networks
-- Paper: https://arxiv.org/abs/1609.02907
-- Code: https://github.com/tkipf/gcn
-"""
-import argparse, time, math
-import numpy as np
-import networkx as nx
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
 import dgl
-from dgl.data import register_data_args
 from dgl.data import CoraGraphDataset, CiteseerGraphDataset, PubmedGraphDataset
-
+import argparse
+import math
 
 def gcn_msg(edge):
     msg = edge.src['h'] * edge.src['norm']
     return {'m': msg}
 
-
 def gcn_reduce(node):
     accum = torch.sum(node.mailbox['m'], 1) * node.data['norm']
     return {'h': accum}
 
-
 class NodeApplyModule(nn.Module):
```

Comment:
Let's try to remove this module.  Bias should have been taken care of by DGL's `GraphConv` module while activation can be in `GCN`'s forward function.

Reply:
Just wanna make sure...
You mean, we can remove NodeApplyModule and GCNLayer module, and just use intrinsic GraphConv module? wouldn't that be the same as train.py implementation? (like you mentioned below, only having one implementation with GraphConv in this folder)
Or you mean, I can use GraphConv in this user-defined gcn implementation, to further simply the script?